cards/location-standard/template.hbs

```diff
@@ -182,7 +182,7 @@
     </div>
   {{/if}}
   <div class="HitchhikerLocationStandard-detailsText js-HitchhikerCard-detailsText{{#if showExcessDetailsToggle}} js-hidden{{/if}}">
-    {{card.details}}
+    {{{card.details}}}
```

Comment:
I think we should leave this as is with two curly braces. Not all cards were meant to support RTF by default.